cli/worker.rs

```diff
@@ -173,6 +189,12 @@ impl CliMainWorker {
         .with_event_loop(coverage_collector.stop_collecting().boxed_local())
         .await?;
     }
+    if let Some(hot_reload_manager) = maybe_hot_reload_manager.as_mut() {
+      self
+        .worker
+        .with_event_loop(hot_reload_manager.stop().boxed_local())
+        .await?;
+    }
```

Comment:
If HMR manager stops here we should still allow for full reload when a file is changed.

Reply:
This is tricky, due to timing. I introduced a "watcher mode" that depending on the variant either reloads whole process or wait for manual restart. The problem is that some programs (eg. `console.log("hello!")`) don't use TLA and thus finishing synchronously during evaluation. I think I might need to change the watcher modes to "hijack" watcher at some point instead of being fully manual.